Settings Page

bhavitaap opened the pull request:
https://linear.app/bread-and-roses/issue/BRE-42/settings-page

created settings cards and hardcoded dummy info

[//]: # "Feel free to customize this template to your project's vibes"

## :bread: What's new in this PR
### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- added settings cards for each of the subsections in the settings page
- hardcoded dummy information (based on the figma)


### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: <img src=https://github.com/user-attachments/assets/d7878cec-31f2-4a83-adb6-a74443133636" width="240" height="540">


## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- app/settings/page.tsx
- components/SettingsCard <- 5 new files created in this folder for each of the subsections
- expected behavior: when navigating to the Settings page, it should look like the Figma 
- feedback: is there possibly a way to collapse the files into one and use props instead of hardcoding the info


## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- update the info from the database
- for bullet point info make it loop through relevant 
<img width="1099" alt="SettingsPageCards" src="https://github.com/user-attachments/assets/675b68d3-4767-479b-87b0-d450967b50c3" />
info instead


## :pancakes: Relevant links
### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'



### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"



CC: @jxmoose

bhavitaap committed:
Settings Page

https://linear.app/bread-and-roses/issue/BRE-42/settings-page

created settings cards and hardcoded dummy info

jxmoose commented on .env.local.save:
this shouldn't be added to git but also not sure if this file should exist in first place lol

jxmoose commented on components/SettingsCard/SettingsCardPerformanceInterest.tsx:
can you change the px to rem! overall should almost always use rem over px. https://nekocalc.com/px-to-rem-converter u can use this website but for smaller ones just divide by 16 and it should be accurate.

jxmoose commented on components/SettingsCard/SettingsCardPerformanceInterest.tsx:
```diff
@@ -0,0 +1,88 @@
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Type of Act
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
can you change the px to rem! overall should almost always use rem over px. https://nekocalc.com/px-to-rem-converter u can use this website but for smaller ones just divide by 16 and it should be accurate.

jxmoose commented on components/SettingsCard/SettingsCardPerformanceInterest.tsx:
```diff
@@ -0,0 +1,88 @@
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Genre
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
^

jxmoose commented on components/SettingsCard/SettingsCardShowPreferences.tsx:
```diff
@@ -0,0 +1,83 @@
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Facility Type
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
^

jxmoose commented on components/SettingsCard/SettingsCardShowPreferences.tsx:
```diff
@@ -0,0 +1,83 @@
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Location Preferences
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
^

jxmoose commented on components/SettingsCard/SettingsCardShowPreferences.tsx:
```diff
@@ -0,0 +1,83 @@
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Preferred Audience
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
^

jxmoose commented on components/SettingsCard/styles.ts:
```diff
@@ -44,21 +44,24 @@ export const Content = styled.main`
   display: flex;
   flex-direction: column;
-  gap: 4px;
+  gap: 24px;
```
^

jxmoose commented on components/SettingsCard/styles.ts:
```diff
@@ -44,21 +44,24 @@ export const Content = styled.main`
 `;
 
+export const SettingDetail = styled.main`
+  gap: 4px;
```
^

jxmoose reviewed (changes requested):
overall looks great! just some small changes, if you look at the figma there some of the text should have bullet points if you could add those. also just general things never use px just use rem instead! also delete the .env.local.save file

bhavitaap committed:
Settings Page

https://linear.app/bread-and-roses/issue/BRE-42/settings-page

updated px to rem and deleted env.local.save

jxmoose commented on components/SettingsCard/SettingsCardPerformanceInterest.tsx:
```diff
@@ -0,0 +1,88 @@
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Type of Act
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
can you change these to not use in line styling and add a style component to styles.ts instead

bhavitaap committed:
Settings Page

https://linear.app/bread-and-roses/issue/BRE-42/settings-page

updated bullet point styling to component instead of inline but indentation is off

bhavitaap committed:
Settings Page

https://linear.app/bread-and-roses/issue/BRE-42/settings-page

updated bullet point styling to component instead of inline but indentation is off

bhavitaap committed:
Settings Page

https://linear.app/bread-and-roses/issue/BRE-42/settings-page

updated bullet point styling to component instead of inline styling

jxmoose committed:
wi

jxmoose committed:
style fix
